src/components/Elements.tsx

```diff
@@ -101,74 +87,49 @@ interface PrivateElementsProps {
  */
 export const Elements: FunctionComponent<ElementsProps> = ({
   stripe: rawStripeProp,
-  options,
+  options: optionsProp,
   children,
 }: PrivateElementsProps) => {
-  const final = React.useRef(false);
-  const isMounted = React.useRef(true);
-  const parsed = React.useMemo(() => parseStripeProp(rawStripeProp), [
-    rawStripeProp,
-  ]);
-  const [ctx, setContext] = React.useState<ElementsContextValue>(() => ({
-    stripe: null,
-    elements: null,
-  }));
-
-  const prevStripe = usePrevious(rawStripeProp);
-  const prevOptions = usePrevious(options);
-  if (prevStripe !== null) {
-    if (prevStripe !== rawStripeProp) {
+  const [inputs, setInputs] = React.useState({ rawStripe: rawStripeProp, options: optionsProp })
```

Comment:
It should be more cleaner as we have state which we can trust (never changed after transition from null value).